controller/include/pid.h

```diff
@@ -40,7 +41,13 @@ inline constexpr int AC = 0;
 inline constexpr int RR = 0;
 inline constexpr int IE = 0;
 
-void pid_execute();
 void pid_init();
 
+// Runs one step of the PID module, which is responsible for spinning up the
+// blower.
+//
+// `params` is the last set of params we received from the GUI.  Puts readings
+// from e.g. the pressure sensor into `readings`.
+void pid_execute(const VentParams &params, SensorReadings *readings);
```

Comment:
Can this just return SensorReadings by value?

Reply:
Below I think it's a bad idea to return GuiStatus by value, because it uses a significant amount of our tiny stack space on the Arduino.  Here it's not as bad, because the struct is a lot smaller.  But since this is an issue in *some places*, I think it's probably better to be consistent.  Anyway SensorReadings might get bigger.